libexec/rtld-elf/rtld_c18n.c

```diff
@@ -858,16 +858,27 @@ _rtld_longjmp(struct jmp_args ret, void *rcsp, void **buf)
 	    get_trusted_frame()));
 }
 
+struct jmp_args _rtld_unw_setcontext_epilogue(struct jmp_args ret, void *rcsp,
```

Comment:
Arguments don't need to be named in the function declaration. So something like
```
struct jmp_args _rtld_unw_setcontext_epilogue(struct jmp_args, void *, void **);
```

Reply:
I put it there intentionally because this is an assembly function, so it just documents what's in the registers.